src/inspect_ai/model/_call_tools.py

```diff
@@ -96,6 +100,17 @@ async def call_tool_task(call: ToolCall) -> tuple[ChatMessageTool, ToolEvent]:
                 if isinstance(ex.filename, str):
                     err = f"{err} Filename '{ex.filename}'."
                 tool_error = ToolCallError("is_a_directory", err)
+            except SandboxOutputLimitExceededError as ex:
+                tool_error = ToolCallError(
+                    "output_limit",
+                    f"The sandbox exec output limit of {ex.limit_str} was exceeded.",
```

Comment:
For the model, do we want to say "The tool output limit of {} was exceeded" (as it doesn't really know about the the "sandbox" but does know about tools)